DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: epireview
 Title: Tools to update and summarise the latest pathogen data from the Pathogen Epidemiology Review Group (PERG)
-Version: 1.1.1
+Version: 1.1.2
```

Comment:
I think this will clash with Sangeetas latest PR (which just got merged in and also had version 1.1.2)

Reply:
Agreed, sorted